src/components/domain-resolver/RedefinedDomainResolver.tsx

```diff
@@ -52,12 +56,16 @@ const RedefinedDomainResolver = (props: RedefinedDomainResolverProps) => {
         ) {
           setError( `This domain is registered but has no records.`)
         } else {
-          setAddresses(response);
+          if (version == actualVersion) {
+            setAddresses(response);
+          }
         }
       } catch (e) {
         setError(e)
```

Comment:
`e.message` ?

Reply:
we processing Error in utils fun getErrorMessage(error?: string | Error)

Reply:
What do you mean?